nengo_loihi/loihi_cx.py

```diff
@@ -780,3 +782,6 @@ def get_probe_output(self, probe):
         x = np.asarray(self.probe_outputs[cx_probe], dtype=np.float32)
         x = x if cx_probe.weights is None else np.dot(x, cx_probe.weights)
         return self._filter_probe(cx_probe, x)
+
+    def close(self):
+        self.closed = True
```

Comment:
I'm not sure if I agree with adding this `close` that does nothing. My problem with it is we haven't done all the other legwork to make sure `CxSimulator` gets closed (when we create it manually in some of our tests). If someone actually adds something to this function down the line, they might not realize all that legwork still needs to be done.

If we want this, I'd add a context manager, make sure it's used in tests, and make sure we error when trying to run a closed simulator. I also might try to free up some of the memory allocated in `build`. I'm happy to do this if others agree.

Reply:
Doesn't seem super high priority to me (we don't expect users to be manually creating `CxSimulator`, and we only do that in one test, so I think allowing `nengo_loihi.Simulator` to handle all the opening and closing is fine).  But doesn't hurt I guess, if you're looking for something to work on 🤷‍♂️